### DISCUSSION TITLE
Drivers compatible with Apple Silicon Chips

### DISCUSSION THREAD

**Discussion by greenmeowpow** [chris1111/Wireless-USB-Big-Sur-Adapter#302] [General]
I wonder if there is an available driver or any plans of creating one compatible with Apple Silicon CHips? I have seen that this driver only works on intel-based macs. I hope there would also be one for M1 and M2.

**Comment by devmni** [comment]
Me too. It seems as it is absolutely impossible to use an USB WiFi stick with an Apple M1. Compiling this project for arm is the last hope.

**Reply by maopareja** [reply]
Me too, I hope that someday it will be possible.

**Comment by alappatsj** [comment]
any update on this?

**Comment by alappatsj** [comment]
> I wonder if there is an available driver or any plans of creating one compatible with Apple Silicon CHips? I have seen that this driver only works on intel-based macs. I hope there would also be one for M1 and M2.

any update on this?
